packages/Liquid-Core.package/LiquidAnswerPanel.class/instance/updateAnswer.st

```diff
@@ -0,0 +1,10 @@
+accessing
+updateAnswer
+|choiceCollection|
+	choiceCollection := self selectedChoices asOrderedCollection.
+	(self question type = 'singleChoice')
+		ifTrue: [(choiceCollection isEmpty) 
+			ifTrue: [self answer votedChoice: nil]
+			ifFalse: [self answer votedChoice: (choiceCollection at: 1)]].
```

Comment:
it seems like we should have two different classes for `SingleChoiceAnswer` and `MultiChoiceAnswer`

Reply:
Maybe I don't quite understand your comment, but we do have two classes `LiquidSingleChoiceAnswer` and `LiquidMultipleChoiceAnswer`.

Reply:
hmm, I might have been a little nit-picky.